content/en/docs/Core concepts/_index.md

```diff
@@ -7,4 +7,58 @@ description: >
   Baker core concepts
 ---
 
-This section explains the core concepts of Baker.
+## Pipeline
+
+A pipeline (a.k.a. Topology) is the configured set of operations that Baker performs during
+its execution.
+
+It is defined by:
+
+* One input component, defining where to fetch records from
+* Zero or more filters, applied sequentially, which together define the **filter chain**. A filter is
+a function that processes record (modifying fields, discarding or creating records).
+* One output component, defining where to send the filtered records to (and which fields)
+* One optional upload component, defining where to send files produced by the output component
+
+Notice that there are two main usage scenarios for Baker, batch or daemon processing, that depend on
+the input component behaviour:
+
+* **Batch processing**: In this case, Baker goes through all the records that are fed
+by the input component, processes them as quickly as possible, and exits when the input component
+ends its job.
+* **Daemon**: in this case, the input component never exits and thus also Baker, that keeps waiting
+for incoming records from the input (e.g.: Kinesis), processes them and sends them to the output.
+
+Also read [Pipeline configuration](/docs/how-to/pipeline_configuration/)
+
+## Record and LogLine
+
+Baker processes objects in form of "records". A Record, in Baker, is an interface that
+provides an abstraction over a record of flattened data, where columns of fields are
+indexed through integers.
+
+Baker currently provides a single implementation of Record, called `LogLine` (
+[API reference](https://pkg.go.dev/github.com/AdRoll/baker#LogLine)).
+
+If `LogLine` doesn't fit your needs, you can [customize the LogLine](/docs/how-to/record_and_logline/)
```

Comment:
```suggestion
If `LogLine` doesn't fit your needs, you can [customize the LogLine struct](/docs/how-to/record_and_logline/)
```

or maybe just

```suggestion
If `LogLine` doesn't fit your needs, you can [customize it](/docs/how-to/record_and_logline/)
```
since LogLine is already at the beggining of the sentence